_events/2020/2020-07-pearc20.md

```diff
@@ -12,19 +12,63 @@ repeated: false
 
 Join us at PEARC20 for a workshop focused on Research Software Engineers.  
 
-The full PEARC20 conference runs July 26-31 and is virtual this year.  Information about the conference is available on the [PEARC20
-conference website](https://pearc.acm.org/pearc20/).  Registration will be required for all participants.
+The **Research Software Engineers Community Workshop** will be July 31st, starting a 8am PDT (Portland, OR time).  It is open to all [PEARC20](https://pearc.acm.org/pearc20/) registrants.  
 
-The **Research Software Engineers Community Workshop** will be July 31st, starting a 8am PDT (Portland, OR time).  It is open to all PEARC20 registrants.  
+The full PEARC20 conference runs July 26-31 and is virtual this year.  Information about the conference is available on the [PEARC20
+conference website](https://pearc.acm.org/pearc20/).  [PEARC20 Registration](http://www.cvent.com/events/practice-and-experience-in-advanced-research-computing-2020-pearc-/event-summary-e4cda8bfe4c545b7bb3e458e13a128c2.aspx?tw=12-7B-8D-67-26-C9-CC-DD-AD-82-E2-F8-E0-D3-05-97) will be required for all workshop participants.
```

Comment:
Do you know what the `?tw=12-7B-8D-67-26-C9-CC-DD-AD-82-E2-F8-E0-D3-05-97` get param is? It seems to work without it, so in case this is a link specific to you (or some other metadata that shouldn't be shared) it might work to just remove it.